tutorials/13_diagnostics_simulation_based_calibration.ipynb

```diff
@@ -557,16 +1061,18 @@
   },
   {
    "cell_type": "markdown",
+   "id": "46b9562d-9c7b-4adf-837e-e9e2ca5abdb6",
    "metadata": {},
    "source": [
     "## A shifted posterior mean\n",
     "\n",
-    "In this scenario we emulate the situation that our posterior estimates incorrectly with a constant shift. We reuse our trained NPE posterior from above and wrap it so that all samples returned expose a constant shift by `+0.1`.\n"
+    "In this scenario, we emulate the situation that our posterior estimates incorrectly with a constant shift. We reuse our trained NPE posterior from above and wrap it so that all samples returned expose a constant shift by `+0.1`.\n"
```

Comment:
If we change the dimensionality we have to train again, no?

Reply:
In the current version, I don't increase the dimensionality. As mentioned [above](https://github.com/sbi-dev/sbi/pull/1051#discussion_r1533605165) I don't increase the dimension after all. I think sticking to the example in the notebook is a good way to not introduce too much mental load of the reader.